action.yml

```diff
@@ -251,8 +254,10 @@ runs:
               if match := re.match(r"Programming Language :: Python :: (\d+\.\d+)$", classifier):
                   version_tokens.append(match.group(1))
 
+          package_name = pkg_info.get("Name")
```

Comment:
I don't think this can realistically happen (build backends should catch this), but if it does, I _think_ we should fail with a helpful error message here?

Reply:
You mean if `Name` is missing. [The spec](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-specifications) does say it's required, but we could probably [emit an error](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-error-message).

Reply:
Yeah that's what I mean. Emitting an error sounds great although I doubt the other tooling will allow us to get as far. :) Always good to add more assertions.

Reply:
d404ab0af268ab3b620596216ff47265da9c4d7f